Cargo.toml

```
@@ -37,12 +37,17 @@ bootloader_api = { version = "0.11.0", path = "api" }
 bootloader-x86_64-common = { version = "0.11.0", path = "common" }
 bootloader-x86_64-bios-common = { version = "0.11.0", path = "bios/common" }
 
+[features]
+default = ["bios", "uefi"]
+bios = ["mbrman", "bootloader_test_runner/bios"]
+uefi = ["gpt", "bootloader_test_runner/uefi"]
```

```suggestion
bios = ["dep:mbrman", "bootloader_test_runner/bios"]
uefi = ["dep:gpt", "bootloader_test_runner/uefi"]
```

We should use the `dep:` syntax to enable the dependencies. Otherwise cargo will create and expose features called `mbrman` and `gpt` for the `bootloader` package.